### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4718] [General]
Some of you know that I live in Florida. I just want to report that the area in which I live was luckily spared from the wrath of Milton. That being said... I will likely not be able to review PRs or respond to Issues until Monday.

**Comment by leigh-pointer** [comment]
Glad to hear you are ok! 
Stay safe

**Comment by mdmontesinos** [comment]
Take your time and be safe!

**Reply by thinkgandhi** [reply]
Stay safe @sbwalker . I thought so .
